examples/example.drush.yml

```diff
@@ -56,6 +56,12 @@ drush:
     # or multiple --include options may be provided. Drush 8 and earlier did
     # a deep search in ~/.drush and /usr/share/drush/commands when loading
     # command files.
+    #
+    # For testing, specify the namespace component in the key. e.g.:
```

Comment:
For testing?

Reply:
The point of the comment is to indicate to users that this feature is intended for use in tests, not for configuring Drush extensions in a site. There's no technical reason to make that limitation; the word "testing" could be removed if desired.